standard/info/commons/info.lua

```diff
@@ -31,6 +31,15 @@ return {
 			hasSpecialTeam = false,
 			allowManual = true,
 		},
+		transfers = {
+			showTeamName = false, -- bool
+			iconModule = nil, -- string?
+			iconParam = 'pos', -- string?
+			platformIcons = false, -- bool
+			positionConvert = nil, -- string?
+			referencesAsTable = false, -- bool
+			syncPlayers = false, -- bool
+		},
```

Comment:
@Rathoz 
those names for the config values okay?
if yes i would start a PR to add them according to current settings in the templates

Reply:
I don't think we need most of them

positionsCovert -> just check if the wiki has data for position conversions
referencesAsTable -> all wikis should move to this, though should be JSON instead of the current way
iconParam -> let's standardize
iconModule -> can we standardize and then determine base on its existence?
showTeamName -> linked to positionCovert + icons I think? Basically has to shorten down the teams when there's more other data
syncPlayers -> assuming no page will break due to runtime, we should always have this on

Reply:
`showTeamName` --> by default it only shows icon, this forces the team name to be displayed too, hence needed
`iconModule` --> can kick due to using opponent stuff for sc/sc2/sg/wc/smash
`iconParam` --> solved for sc/sc2/wc/sg, but smash uses `head` and `teamfortress` uses class
`platformIcons` --> could make it dependent on the extistence of `Module:Platform`
`positionConvert` --> could make it depend on the existence of `Module:PositionName/data
`
`referencesAsTable` --> needs some conversion runs across all wikis if switching to json but okay for me (currently 38 wikis use tref, rest use plain input)
`syncPlayers` --> always on okay for me :)

overview of current values: https://liquipedia.net/starcraft2/User:Hjpalpha/wip2

Reply:
> needs some conversion runs across all wikis if switching to json but okay for me (currently 38 wikis use tref, rest use plain input)

Easiest probably to swap all to "TRef" initially and then when we go live to convert it to json

Reply:
since i use new module names we can just gradually push it wiki by wiki

Reply:
- ref stuff still to be tested (probably later today), but i think it should work like that (using Json)
- `iconModule`, `platformIcons`, `positionConvert` and `syncPlayers` resolved to not need config anymore
- `iconParam`: question if we can convince `smash` and `teamfortress` wikis to switch param
- `showTeamName` imo needs to remain (set for 12 wikis)

Reply:
`Module:Platform` exists on commons, so we can not check for its existence, hence the `platformIcons` is needed

Reply:
fwiw, `commons:Module:Platform/data` is only really used on r6 and simracing, so might move/copy it there?

Reply:
why simracing, they do not even use it

Reply:
@Rathoz rather have `commons:Module:Platform/data` copied to r6 and removed from commons and use that as indicator to show platform icons or should it remain in `Info.config.transfers`?

- aoe, fifa, cod pokemon and crossfire already have overwrites

Reply:
Fine with the former

Reply:
> showed up in global usage of said module

global usage is buggy^^
on the wiki it shows 0 usages

Reply:
@Rathoz 
opinion regarding `iconParam`
you want to force smash (head) and teamfortess (class) to use standardized `pos`?

Reply:
So I was thinking that smash (and for some other games), it might be better to move it from a "position" to a standard character input, similar to factions

Reply:
we could add faction/race/head to commons and also store it as faction
that would solve the need for any customs too